src/bio.h

```diff
@@ -40,11 +40,13 @@ void bioKillThreads(void);
 void bioCreateCloseJob(int fd);
 void bioCreateFsyncJob(int fd);
 void bioCreateLazyFreeJob(lazy_free_fn free_fn, int arg_count, ...);
+void bioCreateFsyncCloseJob(int fd);
 
 /* Background job opcodes */
 #define BIO_CLOSE_FILE    0 /* Deferred close(2) syscall. */
 #define BIO_AOF_FSYNC     1 /* Deferred AOF fsync. */
 #define BIO_LAZY_FREE     2 /* Deferred objects freeing. */
-#define BIO_NUM_OPS       3
+#define BIO_FSYNC_AND_CLOSE 3 /* Deferred fsync and close. */
+#define BIO_NUM_OPS       4
```

Comment:
it now occurs to me that it's a waste to open another thread for this.
i think we should somehow make this an argument for either BIO_AOF_FSYNC, or BIO_CLOSE_FILE.
i see that currently in one place (INFO) the new job is counted with AOF_FSYNC, but in another (aofFsyncInProgress) it isn't.
we'll need to decide which one we prefer.
i lean towards BIO_CLOSE_FILE.

Reply:
> it now occurs to me that it's a waste to open another thread for this.

yes, my first thought was the same, and i was think do it in BIO_AOF_FSYNC, but then somehow i did not raise it, and go to the BIO_FSYNC_AND_CLOSE.

> i lean towards BIO_CLOSE_FILE.

i am also leaning towards BIO_CLOSE_FILE.